docs/source/plugins/rhai.mdx

```diff
@@ -8,6 +8,57 @@ import { Link } from 'gatsby';
 
 TODO:
 
-- Explain the life-cycle ordering (before all other native plugins?)
+- Explain the life-cycle ordering (clarify that it's before all other native plugins?)
 - Give examples
 
+---
+
+If you have specific needs that requires you to write your own plugin but you don't want to learn Rust and you want to use a scripting language, rather than a compiled language, you can use [`rhai`](http://rhai.rs) script to implement some business logic in Router.
+
+The rhai plugin is using the same mechanisms as native Rust plugins and has access to the same life-cycle events.
+
+### Example
+
+#### Configuration
+
+```yaml
+rhai:
+    filename: "location_of_your_rhai_script.rhai"
+```
```

Comment:
let's add a note that there can be only one rhai plugin? (so only one file)

Reply:
Done.